.travis.yml

```diff
@@ -52,12 +51,7 @@ before_install:
   - conda update --yes conda
 
 install:
-  - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then
```

Comment:
@thast: we no longer need this because if was originally checking if we were installing python 2 or python 3. Now, we are only ever installing python 3, so we can follow those download instructions

Reply:
@lheagy : Sorry if I was not clear. My comment above (with screenshot, as I could not comment a line you did not modidy) concerns line 40 and below. It seems that in the "before_install" section we still have a Python 2.7 if statement.

Reply:
ah, good call - thanks @thast. I just pushed an update.